Zend/tests/partial_application/variation_variadics_007.phpt

```diff
@@ -2,7 +2,7 @@
 Partial application variation extra through variadic
 --FILE--
 <?php
-function appliesCb($cb) { return runsCb($cb(1, ?, ?, ...)); } 
```

Comment:
Should probably still keep the old variant and check that it does error?

Reply:
Yeah, okay